docs/userguide/covariates.md

```diff
@@ -117,47 +117,54 @@ Darts' forecasting models accept optional `past_covariates` and / or `future_cov
+[DLinearModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.dlinear.html#darts.models.forecasting.dlinear.DLinearModel) | ✅ | ✅ | ✅ 
+[NLinearModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.nlinear.html#darts.models.forecasting.nlinear.NLinearModel) | ✅ | ✅ | ✅
+[TiDEModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.tide_model.html#darts.models.forecasting.tide_model.TiDEModel) | ✅ | ✅ | ✅
+Ensemble Models (e) | ✅ | ✅ | ✅
 
 **Table 1: Darts' forecasting models and their covariate support**
 
 
-`*` `RegressionModel` including `RandomForest`, `LinearRegressionModel` and `LightGBMModel`. `RegressionModel` is a
-special kind of GFM which can use arbitrary lags on covariates (past and/or future)
-and past targets to do predictions.
+(a) Naive Baselines including [NaiveMean](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.baselines.html#darts.models.forecasting.baselines.NaiveMean), [NaiveSeasonal](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.baselines.html#darts.models.forecasting.baselines.NaiveSeasonal), [NaiveDrift](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.baselines.html#darts.models.forecasting.baselines.NaiveDrift), and [NaiveMovingAverage](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.baselines.html#darts.models.forecasting.baselines.NaiveMovingAverage).
 
-`**` `RNNModel` including `LSTM` and `GRU`; equivalent to DeepAR in its probabilistic version
+(b) Regression Models including [RegressionModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.regression_model.html#regression-model), [LinearRegressionModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.linear_regression_model.html#darts.models.forecasting.linear_regression_model.LinearRegressionModel), [RandomForest](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.random_forest.html#darts.models.forecasting.random_forest.RandomForest), [LightGBMModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.lgbm.html#darts.models.forecasting.lgbm.LightGBMModel), [XGBModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.xgboost.html#darts.models.forecasting.xgboost.XGBModel), and [CatBoostModel](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.catboost_model.html#darts.models.forecasting.catboost_model.CatBoostModel). RegressionModel is a special kind of GFM which can use arbitrary lags on covariates (past and/or future) and past targets to do predictions.
 
-`***` `BlockRNNModel` including `LSTM` and `GRU`
+(c) `RNNModel` including `LSTM` and `GRU`; equivalent to DeepAR in its probabilistic version
+
+(d) `BlockRNNModel` including `LSTM` and `GRU`
```

Comment:
For the sake of consistency, I would move the links for RNNModel and BlockRNNModel in the table to the references (here). And eventually mention that the type of the model in controlled by one parameter?